include/flatbuffers/idl.h

```diff
@@ -908,6 +913,8 @@ class Parser : public ParserState {
   std::string error_;         // User readable error_ if Parse() == false
 
   FlatBufferBuilder builder_;  // any data contained in the file
+  flexbuffers::Builder flex_builder_;
```

Comment:
If this is default constructed it will only share keys. I think either we should default to `BUILDER_FLAG_SHARE_ALL` (prefer smaller binaries at slight runtime cost for flatc) or we should add additional flags to control this (which may not be worth it right now)